components/common/LeftNav.tsx

```diff
@@ -55,6 +61,21 @@ function LeftNav({
     signInWithGoogle,
     signOutAndClear,
 }: LeftNavProps): JSX.Element {
+    const [role, setRole] = useState<Role | undefined>(undefined)
+
+    const handleSignOut = () => {
```

Comment:
If we go a component upwards as described in the component below, we can remove this, no?